packages/get-platform/src/getPlatform.ts

```diff
@@ -55,14 +56,18 @@ export async function getos(): Promise<GetOSResult> {
   }
 
   const distro = await resolveDistro()
+  const archFromUname = await getArchFromUname()
 
-  if (distro === 'musl' && arch !== 'x64') {
-    throw new Error(
-      `Prisma only supports Linux Alpine on the amd64 (x86_64) system architecture. If you're running Prisma on Docker, please use Docker Buildx to simulate the amd64 architecture on your device as explained by this comment: https://github.com/prisma/prisma/issues/8478#issuecomment-1355209706`,
+  // TODO: add 'arm64' to the `[...].includes(arch)` check once we have arm64 engines for Alpine
+  if (distro === 'musl' && !['amd64'].includes(arch)) {
```

Comment:
```suggestion
  if (distro === 'musl' && !['amd64', 'x64'].includes(arch)) {
```

Reply:
Fixed

Reply:
Quick question: Would this have failed with a test somewhere?

Reply:
Currently, no. https://github.com/prisma/ecosystem-tests/issues/3332 would add assertions on some messages in `ecosystem-tests / docker`, we could catch this sort of things by performing assertions on warnings as well.